### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by shizhMSFT**
Update `README.md` for reviewing guide and code of conduct.

Partially resolves the issue https://github.com/oras-project/community/issues/38

### CODE REVIEW

**Review by sammychinedu2ky — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
-# oras-dotnet
-ORAS .NET library
+# ORAS .NET library
+
+> **Warning** This project is currently under active development. The API may and will change incompatibly from one commit to another.
```
Comment (TerryHowe) [inline README.md:3]: To me, active development is the desired state of any project. Maybe "alpha development" or "initial development"

```suggestion
> **Warning** This project is currently under initial development. The API may and will be incompatibly from one commit to another.
```

**Review by TerryHowe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SteveLasker) [reply README.md:3]: @shizhMSFT, can we add some text that describes the scope of this repo/sub-project?

**Review by sajayantony — APPROVED** [review]

**Review by TerryHowe — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (SteveLasker) [reply README.md]: The question about scope of each sub-project continues to come up. I'm suggesting the readme can define the scope:
"The oras-dotnet repo is scoped at implementing a dotnet version of ORAS functionality. "

The interesting thing that came up with the oras-py implementation is whether each library is following an implementation of a common spec? What's the reference for each library? 
For now, It's probably ok to just say the above.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Review by Wwwsylvia — APPROVED** [review]
LGTM

**Review by sammychinedu2ky — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (shizhMSFT) [reply README.md]: @SteveLasker I've created the following issue
- https://github.com/oras-project/community/issues/46

to discuss your concerns.

**Comment by shizhMSFT** [comment]
Merging this PR as it got approved by a majority of org-level maintainers and all conversations are resolved.

**Pull request merged**